The-SMIRNOFF-force-field-format.md

```diff
@@ -218,6 +216,13 @@ For example, to ensure water molecules are assigned partial charges for [TIP3P](
 </LibraryCharges>
 ```
 
+
+| LibraryCharges section tag version | Tag attributes and default values    | Required parameter attributes   | Optional parameter attributes |
+|------------------------------------|--------------------------------------|---------------------------------|-------------------------------|
+| 0.3                                |                                      | `smirks`, `charges` (indexed)   | `name`, `id`, `parent_id`     |
+
+
+
```

Comment:
Yes. Each _section_ has its own version in the SMIRNOFF 0.3 spec. 

This _seems_ excessive, but we recognized during the last spec revamp that, for example, the `Electrostatics` tag might in the future need to contain detailed PME settings, or that maybe some day we'll need to indicate that the `Angles` section can support `partial_bondorder` keywords. If we only have the top-level SMIRNOFF tag recording complete file compatibility, we'd be incrementing the number constantly. So, the top-level SMIRNOFF tag only encodes things about _general_ structure that affects (de)serialization (eg. units in headers vs units in quantity), without worrying about the specific keywords inside. This leaves it up to the specific _section_ versions to indicate changes in keywords or behavior.

Reply:
Confusingly, all _section_ versions begin at 0.3, since that's the first SMIRNOFF spec version that stated that sections have versions. That may have been a poor choice by me, but we did it, so that's the system now.